clients/src/main/resources/common/message/ShareFetchResponse.json

```diff
@@ -0,0 +1,70 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "apiKey": 78,
+  "type": "response",
+  "name": "ShareFetchResponse",
+  "validVersions": "0",
+  "flexibleVersions": "0+",
+  "fields": [
+    { "name": "ThrottleTimeMs", "type": "int32", "versions": "0+", "ignorable": true,
+      "about": "The duration in milliseconds for which the request was throttled due to a quota violation, or zero if the request did not violate any quota." },
+    { "name": "ErrorCode", "type": "int16", "versions": "0+", "ignorable": true,
+      "about": "The top level response error code." },
+    { "name": "Responses", "type": "[]ShareFetchableTopicResponse", "versions": "0+",
+      "about": "The response topics.", "fields": [
+      { "name": "TopicId", "type": "uuid", "versions": "0+", "ignorable": true,
+        "about": "The unique topic ID."},
+      { "name": "Partitions", "type": "[]PartitionData", "versions": "0+",
+        "about": "The topic partitions.", "fields": [
+        { "name": "PartitionIndex", "type": "int32", "versions": "0+",
+          "about": "The partition index." },
+        { "name": "ErrorCode", "type": "int16", "versions": "0+",
+          "about": "The error code, or 0 if there was no fetch error." },
+        { "name": "AcknowledgeErrorCode", "type": "int16", "versions": "0+",
```

Comment:
Minor: KIP needs to remove extra `"` in this line.

Reply:
Also why do we need `AcknowledgeErrorCode` in ShareFetch?

Reply:
Because if `ShareFetchRequest` includes an acknowledge request, we can fail the acknowledge but not the fetch. Also, please edit the KIP to match if there are typos that need to be corrected.

Reply:
Sure, doing them.